.github/workflows/release.yml

```diff
@@ -0,0 +1,88 @@
+name: release-muggle
+
+on:
+  push:
+    tags:
+      - '*.*.*'
+      - '*.*.*-*'
```

Comment:
For now, I think we should filter this by branch. Check this: https://github.com/chaoss/grimoirelab-sortinghat/blob/muggle/.github/workflows/tests.yml

Reply:
Apparently, if we add the branch, it will create a release every time there's a commit to a branch. We have been checking and this workflow will only work for commits in muggle branch.